chromedriver_autoinstaller/utils.py

```diff
@@ -163,12 +163,16 @@ def get_chrome_version():
     elif platform == "win":
         # check both of Program Files and Program Files (x86).
         # if the version isn't found on both of them, version is an empty string.
```

Comment:
```suggestion
```
I don't think we need the comments anymore if we are adding the other suggested changes.